packages/core/src/routes/connector.test.ts

```diff
@@ -188,6 +189,39 @@ describe('connector route', () => {
       });
       expect(response).toHaveProperty('statusCode', 422);
     });
+
+    it('should post a new record when add more than 1 instance with connector factory', async () => {
```

Comment:
This belongs to another PR?

Reply:
Alright, this "enabled" check is now on "post" api, but this description is confusing